src/containers/App.tsx

```diff
@@ -20,8 +20,17 @@ function App() {
     <ThemeProvider theme={Theme}>
       <Switch>
         <Route exact path="/events" component={HomeLandingPage} />
-        <Route exact path="/events/new" component={EventCreationPage} />
+        <Route
+          exact
+          path="/events/new"
+          component={(props) => <EventCreationPage mode="new" {...props} />}
+        />
         <Route exact path="/events/:eventId" component={EventDashboardPage} />
+        <Route
+          exact
+          path="/events/edit/:eventId"
```

Comment:
we wanna follow this pattern instead
```suggestion
          path="/events/:eventId/edit"
```

Reply:
I'll make this change